api/src/main/java/org/openmrs/module/mksreports/MKSReportsConstants.java

```diff
@@ -27,5 +27,9 @@
 	
 	public static final String COMPONENT_REPORTMANAGER_OPDCONSULT = MODULE_ARTIFACT_ID + ".outpatientConsultation";
 	
-	public static final String COMPONENT_REPORTMANAGER_PATIENTHISTORY = MODULE_ARTIFACT_ID + ".patientHistory";
+	public static final String PATIENTHISTORY_ID = "patientHistory";
```

Comment:
Why the renaming? The original name was in line with existing ones.

Reply:
That's not renaming, this diff doesn't show the line it shows as removed is still present below the new line that was added. Please view the whole file to see the complete change where `COMPONENT_REPORTMANAGER_PATIENTHISTORY` is defined as `MODULE_ARTIFACT_ID + "." + PATIENTHISTORY_ID;`